Defer Reporting API content to the new article (#6862)
GoogleChrome/web.dev: src/site/content/en/blog/coop-coep/index.md

### Observe issues using the Reporting API

The [Reporting API](/reporting-api) is another mechanism through which you can
detect various issues. You can configure the Reporting API to instruct your
users' browser to send a report whenever COEP blocks the loading of a resource

> or COOP isolates a popup window. Chrome has supported the

Reviewer: ```suggestion
or COOP isolates a pop-up window. Chrome has supported the
```

> or COOP isolates a pop-up window. Chrome has supported the

Reviewer 2: How about making this sentence more generic?

"Chrome has supported the Reporting API since version 69 for a variety of uses including COEP and COOP."

Author: SGTM. Included your suggestion.

[`Report-To`](https://developer.mozilla.org/docs/Web/HTTP/Headers/Content-Security-Policy/report-to)
header since version 69 for a variety of uses including COEP and COOP.

{% Aside %}

Are you already using the Reporting API with the `Report-To` header? Chrome is

> transitioning to a new version of the Reporting API, which replaces `Report-to` with

Reviewer 2: ```suggestion
transitioning to a new version of the Reporting API, which replaces `Report-To` with
```

> Are you already using the Reporting API with the `Report-To` header? Chrome is
> transitioning to a new version of the Reporting API, based on
> `Reporting-Endpoints`; consider migrating to this new version. Head over
> [Migrate to Reporting API v1](/reporting-api-migration) for details.

Reviewer: ```suggestion
Are you already using the Reporting API with the `Report-To` header? Chrome is
transitioning to a new version of the Reporting API, which replaces `Report-to` with
`Reporting-Endpoints`; consider migrating to the new version. Check out
[Migrate to Reporting API v1](/reporting-api-migration) for details.
```

{% endAside %}

> To learn how to set up a server to receive reports, see [Decide where reports should

Reviewer 2: Small nit, because developers need to do more than server setup:

_To learn how to configure the Reporting API and a set up a server to receive reports, head over to [Using the Reporting API](https://web.dev/reporting-api/#using-the-reporting-api)_

Or simpler

_To learn how to use the Reporting API in your site, head over to [Using the Reporting API](https://web.dev/reporting-api/#using-the-reporting-api)_

> Learn how to set up a server to receive reports at [Decide where reports should
> be sent](/reporting-api/#decide-where-reports-should-be-sent).

Reviewer: ```suggestion
To learn how to set up a server to receive reports, see [Decide where reports should
be sent](/reporting-api/#decide-where-reports-should-be-sent).
```

eventually](https://github.com/whatwg/html/pull/5848).
{% endAside %}

#### Example COOP report

An example [COOP
report](https://html.spec.whatwg.org/multipage/origin.html#reporting) payload

> when a popup window is opened isolated looks like this:

Reviewer: ```suggestion
when a pop-up window is opened isolated looks like this:
```
